packages/flutter_tools/test/general.shard/runner/flutter_command_runner_test.dart

```diff
@@ -144,14 +154,25 @@ void main() {
         final FakeFlutterVersion version = globals.flutterVersion as FakeFlutterVersion;
 
         await runner.run(<String>['--version']);
-
         expect(version.didFetchTagsAndUpdate, true);
+        expect(testUsage.commands, contains(
+          const TestUsageCommand('version'),
+        ));
+        expect(fakeAnalytics.sentEvents, contains(
+          Event.flutterCommandResult(
+              commandPath: 'version',
+              result: 'success',
+              commandHasTerminal: false,
+            )
+        ));
```

Comment:
Adding test cases for both legacy analytics and `package:unified_analytics`... we were previously not testing for the workflow when running `flutter --version`